kafl_fuzzer/gui/__init__.py

```diff
@@ -806,13 +806,13 @@ def main(stdscr):
     gui = GuiDrawer(sys.argv[1], stdscr)
     gui.loop()
 
-if __name__ == "__main__":
+
+def start(args: Namespace):
 
     locale.setlocale(locale.LC_ALL, '')
-    code = locale.getpreferredencoding()
 
-    if len(sys.argv) < 2 or not os.path.exists(sys.argv[1]):
-        print("Usage:\n\t" + sys.argv[0] + " <kafl-workdir>\n")
+    if not args.work_dir:
```

Comment:
this check is redundant / wrong now?

Reply:
the work_dir's existence isn't checked by dynaconf, as it's kAFL responsability to create / purge it when the fuzzer command is invoked.

So dynaconf can't take that responsability because it doesn't have that information.

Reply:
Right, I forgot..